src/date-picker/date-picker-inline.jsx

```diff
@@ -0,0 +1,62 @@
+const React = require('react');
+const Paper = require('../paper');
```

Comment:
Yeah, that's a good idea.
Would we add the click away feature?
I feel like we should.